libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/NavigationUtils.kt

```diff
@@ -0,0 +1,39 @@
+package com.mapbox.navigation.core.telemetry
+
+import android.content.Context
+import android.media.AudioManager
+import android.provider.Settings
+import com.mapbox.navigation.core.telemetry.audio.AudioTypeChain
+import kotlin.math.floor
+
+internal object NavigationUtils {
```

Comment:
Should this be moved to `libnavigation-util` module?

Reply:
also would be great have it as extensions

Reply:
👍 Moved to libnavigation-utils, but not as extension function. They're classless functions now.